### ISSUE TITLE
Unable to view binlog - VS Win 16.10 (MSBuild 16.10)

### ISSUE THREAD

**Issue by mrward** [dotnet/project-system-tools#390]
Creating a classic .NET console project (or a .NET Core console project) and it is not possible to view the binlog - errors are reported:

![image](https://user-images.githubusercontent.com/372361/119704156-38d35580-be4f-11eb-95cc-b165eba40fee.png)

Version installed: 1.0.1.2010501

I suspect this may be due to binlog file changes made in MSBuild 16.10 since I am using the project system tools code in VS Mac and that works OK for MSBuild 16.9 binlogs but not for MSBuild 16.10 binlogs (and has the same error output).

However the errors seem similar to this other issue - https://github.com/dotnet/project-system-tools/issues/388

**Comment by mrward** [comment]
Removing the evaluationId check seems to fix the problem for me on VS Mac.

```
diff --git a/src/LogModel/Builder/ModelBuilder.cs b/src/LogModel/Builder/ModelBuilder.cs
index 58175e5..d90a84a 100644
--- a/src/LogModel/Builder/ModelBuilder.cs
+++ b/src/LogModel/Builder/ModelBuilder.cs
@@ -231,8 +231,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.LogModel.Builder
         private static void CheckProjectEventContext(BuildEventArgs args)
         {
             if (args.BuildEventContext.TargetId != -1 ||
-                args.BuildEventContext.TaskId != -1 ||
-                args.BuildEventContext.EvaluationId != -1)
+                args.BuildEventContext.TaskId != -1)
             {
                 throw new LoggerException(Resources.BadState);
             }

```

Which seems to be the same problem as in https://github.com/dotnet/project-system-tools/issues/388 looking at the diagnosis section.

**Comment by drewnoakes** [comment]
Fixed in #402. Thanks for your work identifying the issue @mrward!
